styles.css

```diff
@@ -0,0 +1,158 @@
+* {
+  margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+body {
+  background-color: #f4f5f6;
+  font-family: 'Slabo 27px', serif;
+}
+
+.container {
+  background-color: white;
+  max-width: 1310px;
+}
+
+.bottom-nav {
+  border-bottom: 2px solid #f4f5f6;
+}
+
+.category,
+.category-2,
+.category-3,
+.category-4,
+.category-5,
+.category-6,
+.category-7 {
+  font-family: 'Roboto Condensed', sans-serif;
+  color: #f72210;
+  text-transform: uppercase;
+  background-color: rgba(255, 255, 255, 0.9);
+  bottom: 12.2em;
+}
+
+.category-3,
+.category-4 {
+  bottom: 9.2em;
+}
+
+.category-4,
+.category-5 {
+  color: #3581f4;
+}
+
+.category-5,
+.category-7 {
+  bottom: 12.2em;
+}
+
+.category-6 {
+  bottom: 5.9em;
+}
+
+.vs-img {
+  display: flex;
+  font: bold 13px/20px 'Roboto condensed', Arial, sans-serif;
+  padding: 8px 15px 9px;
+  text-transform: uppercase;
+  bottom: -20px;
+  z-index: 1;
+}
+
+.vs-line {
+  margin-bottom: -5px;
+  margin-top: -15px;
+}
+
+.see-all {
+  margin-top: -1.3rem !important;
+  margin-left: -1.3rem !important;
+  margin-right: -1.3rem !important;
+  padding: 1rem;
+  margin-bottom: -5px !important;
+}
+
+.periscope-culture {
+  margin-top: 1.5rem;
+  margin-left: -0.6rem !important;
+  margin-right: -0.6rem !important;
+}
+
+.periscope-culture img {
+  width: auto;
+}
+
+.features2 img {
+  width: auto;
+}
+
+.editors-pick {
+  margin-right: -1.5rem !important;
+}
+
+.editors2 {
+  margin-right: -3rem !important;
+  margin-top: 2rem !important;
+}
+
+.the-end {
+  position: relative;
+  left: -120px;
+  top: 18px;
+}
+
+.in-depth {
+  margin-left: -6rem !important;
+}
+
+.text-position {
+  margin-left: -1.1rem !important;
+}
+
+.skew {
+  position: relative;
+  top: 4.3rem;
+  right: -1em;
+  width: 60px;
+  height: 25px;
+  clip-path: polygon(0 0, 100% 0%, 100% 100%, 20% 100%);
+}
+
+@media only screen and (max-width: 992px) {
+  .category,
+  .category-2,
+  .category-3,
+  .category-4,
+  .category-5,
+  .category-6,
+  .category-7 {
+    font: 'Roboto Condensed', sans-serif;
+    color: #f72210;
+    text-transform: uppercase;
```

Comment:
[OPTIONAL]- You can use the bootstrap class `text-uppercase` for this. Feel free to use [this cheatsheet](https://hackerthemes.com/bootstrap-cheatsheet/) for more information.